mypy/plugins/attrs.py

```diff
@@ -1060,3 +1060,43 @@ def evolve_function_sig_callback(ctx: mypy.plugin.FunctionSigContext) -> Callabl
         fallback=ctx.default_signature.fallback,
         name=f"{ctx.default_signature.name} of {inst_type_str}",
     )
+
+
+def fields_function_sig_callback(ctx: mypy.plugin.FunctionSigContext) -> CallableType:
+    """Provide the proper signature for `attrs.fields`."""
```

Comment:
nit - don't think "proper" adds anything here, we obviously don't want to provide an "improper" one :)
```suggestion
    """Provides the signature for `attrs.fields`."""
```

Reply:
Sure. @JelleZijlstra which style of docstrings is preferred here, `Provides x...` or `Provide x...`? The docstrings in the `attrs.py` file look mixed.

Reply:
Not sure, I'd prefer "Provide".